application-templates/starter/src/components/welcome/welcome.jsx

```diff
@@ -7,7 +7,7 @@ import { AngleRightIcon } from '@commercetools-uikit/icons';
 import Spacings from '@commercetools-uikit/spacings';
 import Text from '@commercetools-uikit/text';
 import messages from './messages';
-import styles from './welcome.mod.css';
+import styles from "./welcome.module.css";
```

Comment:
Maybe we should keep single quotes here.

Reply:
Fixed here: https://github.com/commercetools/merchant-center-application-kit/pull/2693/commits/acef0769b03e4723df46a357538fc38d9dd4a71e

Lint staged didn't include `.jsx` file extensions for running prettier.